### PR TITLE
First iteration of example app docker-compose

### PR DESCRIPTION
**Pull request by jnation3406**
The images will need to be changed to our current dockerhub versions once those branches are merged. For now, I included ones on our local dockerhub in case you are interested in running it locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+* `example_requests` - This command creates a set of RequestGroups with different observation types and states, and a set of associated Observations for the Requests that should have been attempted. It requires that a valid User and Proposal be passed in. Requests are created both in the past and in the future of the current time. Targets are choosen randomly out of a predefined list if they are visible given the defined sites in Configdb.
+
+## Running the example
+
+First, you must ensure that you pulled down the submodule `docker-postgresql-multiple-databases`. This can be done when cloning the repo by adding `--recursive-submodules --remote-submodules` to your `git clone` command. It could also be done after cloning the base repo by running `git submodule init` followed by `git submodule update` in the base repo directory. After doing either of these, your `docker-postgresql-multiple-databases` directory should contain a file named `create-multiple-postgresql-databases.sh`, which is used to create multiple databases in the same postgresql container.
```
Comment (markBowman) [inline README.md]: I think --recursive-submodules should be --recurse-submodules

**Review by markBowman — APPROVED** [review]
Joy!

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            - DB_HOST=db
+            - DB_NAME=downtime
+            - DB_USER=postgres
+            - DB_PASS=
+            - SECRET_KEY=ocs_example_downtime_secret_key
+        mem_limit: "512m"
+        restart: always
+        command: >
+            sh -c "python manage.py migrate
+            && python manage.py shell -c \"from django.contrib.auth.models import User; User.objects.create_superuser('test_user', 'test_user@fake.com', 'test_pass')\"
```
Comment (eheinrich) [inline docker-compose.yml:63]: Why didn't the django `create_superuser` management command work here?

**Review by eheinrich — APPROVED** [review]
Nice!

File: docker-compose.yml (hunk 2 above)
Comment (jnation3406) [reply docker-compose.yml:63]: The create_superuser management command has no options to set a password non-interactively, it only prompts for it. This looked to be the best way to create one with a password programmatically.

**Pull request merged**